guide/the-3d-window.md

```diff
@@ -2,15 +2,25 @@
 
 ### Selecting an Object
 
-A single mouse click allows you to select a solid object.
+A single mouse left click allows you to select a [Solid](../reference/solid.md) object.
 The bounding object of a selected solid is represented by white lines.
 These lines turn pink if the solid is colliding with another one and blue when the solid is idle, i.e., it comes to rest and it does not interact with any other active solid.
-Selecting a robot enables the `Show Robot Window` item in the `Tools` menu.
-Double-clicking on a solid object opens the Robot Window and if it was closed, the Scene Tree.
+Double-clicking on a [Robot](../reference/robot.md) opens the Robot Window.
 
 If an object has a solid subpart, then it is also possible to select only this subpart by clicking on it once the whole object is already selected, or by clicking on it while holding down the Alt key.
 Linux users should also hold down the Control key (Ctrl) together with the Alt key.
 
+#### Context Menu
+
+%figure "The context menu allows you to perform various actions depending on the current selected object"
+
+![context_menu.png](images/context_menu.png)
+
+%end
+
+When an object is selected in the 3D Window is is possible to right click on it to open the context menu.
```

Comment:
An object doesn't need to be selected to open the context menu.
It is the right-click action that picks the object and select it before opening its context menu.

In the rest of the page we use `3D window`

typo: "is is possible" -> "it is possible"